static/js/src/contact-forms.js

```diff
@@ -0,0 +1,49 @@
+function validateCheckbox(event, fieldsetId) {
+    const checkboxes = Array.from(document.getElementById(fieldsetId).querySelectorAll("input[class='p-checkbox__input']"));
+    if (event.currentTarget.checked) {
+        checkboxes[0].removeAttribute("required");
+    }
+}
+
+function getRadioItemValue(fieldset) {
+    const selectedRadio = fieldset.querySelector("input[name='how-many-machines-do-you-have']:checked")
```

Comment:
I don't know if you wrote this piece of code, but it seems uncessary? Duplication of line `42`:

```
const inputs = howManyMachinesFieldset?.querySelectorAll("input[name='how-many-machines-do-you-have']")
```

Reply:
They are not, first one checks for the selected values as the field is required, second selects all inputs